mne/preprocessing/interpolate.py

```diff
@@ -111,35 +115,64 @@ def interpolate_bridged_electrodes(inst, bridged_idx):
     bads_orig = inst.info['bads']
     inst.info['bads'] = list()
 
+    # look for group of bad channels
+    nodes = sorted(set(chain(*bridged_idx)))
+    G_dense = np.zeros((len(nodes), len(nodes)))
+    # fill the edges with a weight of 1
+    for bridge in bridged_idx:
+        idx0 = nodes.index(bridge[0])
+        idx1 = nodes.index(bridge[1])
+        G_dense[idx0, idx1] = 1
+        G_dense[idx1, idx0] = 1
+    # look for connected components
+    _, labels = connected_components(csr_matrix(G_dense), directed=False)
+    groups_idx = [
+        [nodes[j] for j in np.where(labels == k)[0]] for k in set(labels)
+    ]
+    groups_names = [
+        [inst.info.ch_names[k] for k in group_idx] for group_idx in groups_idx
+    ]
+
+    # warn for all bridged areas that include too many electrodes
+    for group_names in groups_names:
+        if len(group_names) > 4:
+            warn(f"The channels {', '.join(group_names)} are bridged together "
+                 "and form a large area of bridged electrodes. Interpolation "
+                 "might be inaccurate.")
```

Comment:
I chose to emit the warning from N=5 onwards. Maybe it's a bit too lenient. LMK. 
Also, maybe we could improve a bit the warning, what would be a safe course of action we could recommend when you have a large bridged area? I would drop the electrodes, or at least most of the electrodes.

Reply:
You could add a `bad_limit=4` param. It makes it explicit to people that there is a limit they should think about. And I'd maybe an error if it's exceeded, since they can work around it by setting it higher themselves

Reply:
Sounds good, I'll add it.